src/index.js

```diff
@@ -0,0 +1,140 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Hi folks, could one or both of you tell me more about the design decision to wrap all of the Javascript in an anonymous function attached to the `DOMContentLoaded` event?